holidays/observed_holiday_base.py

```diff
@@ -74,10 +74,12 @@ def __add__(self, other):
 THU_FRI_TO_NEXT_MON = ObservedRule({THU: +4, FRI: +3})
 THU_FRI_TO_NEXT_WORKDAY = ObservedRule({THU: +7, FRI: +7})
 THU_FRI_SUN_TO_NEXT_MON = ObservedRule({THU: +4, FRI: +3, SUN: +1})
+THU_FRI_SAT_TO_NEXT_MON_TUE = ObservedRule({THU: +4, FRI: +3, SAT: +3})
 
 FRI_SAT_TO_NEXT_WORKDAY = ObservedRule({FRI: +7, SAT: +7})
 FRI_SUN_TO_NEXT_MON = ObservedRule({FRI: +3, SUN: +1})
 FRI_SUN_TO_NEXT_SAT_MON = ObservedRule({FRI: +1, SUN: +1})
+FRI_SAT_SUN_TO_NEXT_TUE_WED = ObservedRule({FRI: +4, SAT: +4, SUN: +3})
```

Comment:
And `FRI_TO_NEXT_TUE`  + `SAT_SUN_TO_NEXT_WED` (both need to be added) here

This will make the observance rules clear and unambiguous.